atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/timestamp/InDbTimestampBoundStore.java

```diff
@@ -197,8 +198,20 @@ private void createLimit(Connection connection, long limit) throws SQLException
 
     private void createTimestampTable(Connection connection) throws SQLException {
         try (Statement statement = connection.createStatement()) {
-            statement.execute(String.format("CREATE TABLE IF NOT EXISTS %s ( last_allocated int8 NOT NULL )",
-                    timestampTable.getQualifiedName()));
+            if (getDbType(connection).equals(DBType.ORACLE)) {
```

Comment:
If we do stick to using an `if`, we should refactor the contents of the each branch into methods with a `Postgres`/`Oracle` suffix like in `DbKvsGetRanges`